java/trace/src/test/scala/org/typelevel/otel4s/java/trace/TracerSuite.scala

```diff
@@ -472,6 +471,7 @@ class TracerSuite extends CatsEffectSuite {
     }
   }
 
+  /*
```

Comment:
We've lost the ability to trace streams in this API.  I believe I've made a `Local[Stream[F, *], E]`, so we should be able to get it back, but this PR is already getting large.